shiksha-migration/cohort-academicYear-userdb-migration.js

```
@@ -0,0 +1,185 @@
+          `${apiBaseUrl}/user/v1/cohort-academic-year/create`,
+          {
+            cohortId: cohort.cohortId,
+            academicYearId: academicYearId,
+            createdBy: adminUserId,
+            updatedBy: adminUserId,
+          },
+          {
+            headers: {
+              tenantId: tenantId,
+              'Content-Type': 'application/json',
+              'Authorization': `Bearer ${bearerToken}`,
```

ESLint

[error] Replace `'Authorization'` with `Authorization`

(prettier/prettier)